website/docs/customization/plugin/plugin_only.md

```diff
@@ -0,0 +1,70 @@
+---
+id: plugin_only
+description: The Plugin Only Mode
+slug: /plugin/plugin_only
+---
+
+# The Plugin-Only Mode
+
+The plugin-only mode is a restricted mode of TaskWeaver that only allows you to use plugins. 
+Compared to the full mode, the plugin-only mode has the following restrictions:
+
+1. The generated code only contains the calls to the plugins. 
+   For example, the following code only calls the `ascii_render` plugin and does not contain any "free-form" code.
+    ```python
+    r1=ascii_render(text="Hello")
+    r1
+    ```
+    
+2. Only the plugins with `plugin_only: true` in the yaml file will be loaded. 
+   For example, the following plugin will be loaded in the plugin-only mode:
+    ```yaml
+    name: ascii_render
+    code: ascii_render
+    plugin_only: true
+    ...
+    ```
+   If this field is not specified, the default value is `false`.
+   For these plugins, the argument type can only be `str`, `int`, `boolean`, or `float`.
+   Other types such as `DataFrame` are not allowed.
+   Essentially, we consider these plugins only produce "text-like" output that can be directly consumed by the LLM.
+
+To enable the plugin-only mode, you can add the configuration `"session.plugin_only_mode": true` 
+in the project configuration file `taskweaver_config.json`.
+
+## Why do we need the plugin-only mode?
```

Comment:
Better put it in an earlier position? first explain motivation?

Reply:
But people don't know what plugin-only mode actually means

Reply:
that's why we need first explain why we need plugin-only mode rather than tell them how to configure it